### DISCUSSION TITLE
[PSD6] Esercizio Slide 8

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#149] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210017147-fbc9abcc-134a-4f35-b784-1250d2cfc58d.png)

**Comment by Elia-Belli** [comment]
- A B C Y
0 0 0 0 
0 0 1 0
0 1 0 0
0 1 1 1
1 0 0 0
1 0 1 1
1 1 0 1
1 1 1 1

- **SOP minima**: $BC + AB + AC$
- **POS minima**: $(C+A)(C+B)(B+A)$

- $\bar A B C + A(B+C) = BC +AB +AC$
$\bar A B C + AB + AC = BC + AB + AC$
$B(\bar A C+ A) + AC = BC + AB + AC$
$B((\bar A + A)(A+C))+AC = BC + AB + AC$
$AB + BC + AC = BC + AB + AC$

**Comment by Emanuele-Dambrosio** [comment]
TT: OK
SOP e POS: hai dimenticato i negati
E) Non capisco da dove tira fuori quel !A nella semplificazione, comunque anche a me esce la tua soluzione per la semplificazione che coinciderebbero quindi con quella proposta, se non fosse per quel'!A

**Comment by Elia-Belli** [comment]
@Emanuele-Dambrosio non ho dimenticato i negati, così è già minimizzata (puoi fare un controllo con le K-map)
E) $\bar A$ si trova nel primo implicante, raccogliere $B$ e applicare la proprietà distributiva fa tornare quei passaggi

**Comment by FeddyLix17** [comment]
tutto corretto
